src/components/DatasetItem.vue

```diff
@@ -56,6 +73,9 @@
               title="Filter by this lab"
               @click="addFilter('institution')"></span>
       </div>
+      <div v-if="dataset.status != 'STARTED'">
```

Comment:
Will `QUEUED` datasets also match this condition? The same question is valid for the nodes above